functions/Install-DbaInstance.ps1

```diff
@@ -290,7 +290,19 @@ function Install-DbaInstance {
                 $output += "[$key]"
                 if ($Content.$key -is [hashtable]) {
                     foreach ($sectionKey in $Content.$key.Keys) {
-                        $output += "$sectionKey=`"$($Content.$key.$sectionKey -join ',')`""
+                        $origVal = $Content.$key.$sectionKey
+                        if ($origVal -is [array]) {
+                            $output += "$sectionKey=`"$($origVal -join ',')`""
+                        } else {
+                            if ($origVal -is [int]) {
+                                $origVal = "$origVal"
+                            }
+                            if ($origVal -ne $origVal.Trim('"')) {
+                                $output += "$sectionKey=`"$($origVal.Trim('"') -join ',')`""
+                            } else {
+                                $output += "$sectionKey==$(($origVal | ForEach-Object { '"{0}"' -f $_  }) -join ' ')"
```

Comment:
Here again, this is not an array, hence no need for ForEach-Object. And this one should be quoted, if I understand the `if` correctly. Also, two equal signs 🤷‍♂ 
```suggestion
                                $output += "$sectionKey=`"$origVal`""
```

Reply:
agreed, same thing, cleaner code